src/send.jl

```diff
@@ -43,3 +43,27 @@ function sendattachmenttoslack(data, user_endpoint::String)
 
     return (String(r.body))
 end
+
+"""
+    sendfiletoslack(filepath, channel, token)
```

Comment:
I unified the naming convention with that of other functions such as `sendtoslack`, but I think just `sendfile` is fine. (And also, I thought it would be better to change `sendtoslack` -> `sendtext`, `sendattachmenttoslack`->`sendattachment`.)